typeid/typeid-sql/sql/03_typeid.sql

```diff
@@ -13,8 +13,8 @@ create or replace function typeid_generate(prefix text)
 returns typeid
 as $$
 begin
-  if (prefix is null) or not (prefix ~ '^[a-z]{0,63}$') then
-    raise exception 'typeid prefix must match the regular expression [a-z]{0,63}';
+  if (prefix is null) or not (prefix ~ '^[a-z_]{0,63}$') then
```

Comment:
Technically this is not the right regex expression for two reasons:
1. It allows underscores at the beginning
2. It allows underscores at the end.

But underscores should only be allowed in the middle of the prefix. See https://github.com/jetify-com/typeid/tree/main/spec#type-prefix

The regex suggested there is `^([a-z]([a-z_]{0,61}[a-z])?)?$`